devops/actions/llvm_test_suite/action.yml

```diff
@@ -84,11 +84,16 @@ runs:
       export LIT_OPTS="-v --no-progress-bar --show-unsupported --time-tests -o $PWD/build/results_${{ inputs.results_name_suffix }}.json"
       if [ -e /runtimes/oneapi-tbb/env/vars.sh ]; then
         source /runtimes/oneapi-tbb/env/vars.sh;
+      elif [ -e /opt/runtimes/oneapi-tbb/env/vars.sh ]; then
+        source /opt/runtimes/oneapi-tbb/env/vars.sh;
+      else
+        echo "no TBB vars in /opt/runtimes or /runtimes";
       fi
       # TODO remove workaround of FPGA emu bug
       mkdir -p icd
       echo /usr/lib/x86_64-linux-gnu/intel-opencl/libigdrcl.so > icd/gpu.icd
       echo /runtimes/oclcpu/x64/libintelocl.so > icd/cpu.icd
+      echo /opt/runtimes/oclcpu/x64/libintelocl.so > icd/cpu2.icd
```

Comment:
Is it going to produce 2 CPU devices on some systems?

Reply:
No. This is only in our docker images.   The path is either/or, never both.  So while we make two different .icd files, only one of them will work. The other will be ignored (which I've tested and confirmed is harmless).